tensorflow/lite/tools/optimize/quantize_model_test.cc

```diff
@@ -1001,6 +1001,7 @@ TEST_F(QuantizeMultiInputAddWithReshapeTest, VerifyAddQuantization) {
 class QuantizeConstInputTest : public QuantizeModelTest {
  protected:
   QuantizeConstInputTest() {
+    tensor_type_ = GetParam();
```

Comment:
getting this error internally , can you please check ?

`error: use of undeclared identifier 'tensor_type_'`

Reply:
Sorry, I forgot... this is dependent on (a fix for) another PR, namely: https://github.com/tensorflow/tensorflow/pull/42373